__tests__/app.test.js

```diff
@@ -155,4 +155,64 @@ describe("/api/articles/:article_id/comments", () => {
                 expect(body.msg).toBe("article not found");
             });
     });
-});
\ No newline at end of file
+    test("POST:201 should respond with the posted comment", () => {
+        const timestamp = (new Date).toString();
+        const newComment = {
+            username: 'icellusedkars',
+            body: 'what a crazy comment I just left lol'
+        };
+        const expectedResponse = {
+            comment_id: 19,
+            votes: 0,
+            author: 'icellusedkars',
+            body: 'what a crazy comment I just left lol',
+            article_id: 2
+        }
+        return request(app)
+            .post("/api/articles/2/comments")
+            .send(newComment)
+            .expect(201)
+            .then(({ body }) => {
+                expect(body.comment).toMatchObject(expectedResponse);
+                expect(new Date(body.comment.created_at).toString()).toBe(timestamp);
+            });
```

Comment:
Could we get another test here for what would happen if we add unnecessary properties? I.e. if we add something other than username or body as well as those properties?